app/src/test/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModelTest.kt

```diff
@@ -157,6 +172,161 @@ class TransactionDetailsViewModelTest {
                 CHAIN_ID,
                 "tx_details_id"
             )
+            transactionLocalRepository.updateLocalTx(any(), any<String>())
```

Comment:
To be honest this is not a good verification. We should know exactly what values it is being called with. This is not something blocking the PR anymore, but this is not the test quality I would expect for critical flows.